### PR TITLE
added a caption and improved the radio buttons descriptions

### LINKED ISSUE #1794: Better description of notification options
Capture d’écran_2019-05-09_08-53-15

### PR DESCRIPTION
**Pull request by ma12-co**
Fix #1794

### CODE REVIEW

File: js/views/templates/roomlistview.handlebars [hunk 1]
```diff
@@ -43,7 +43,9 @@
 				<span>{{copyLinkText}}</span>
 			</button>
 		</li>
-		<li><div class="separator"></div></li>
+		<li><div class="separator"></div>
+		<li class="app-navigation-entry-menu-caption">Chat notifications</li>
```
Comment (nickvergessen) [inline js/views/templates/roomlistview.handlebars]: Please use a placeholder similar to the other texts, so they can be translated:

```suggestion
		<li class="app-navigation-entry-menu-caption">{{notificationCaptionText}}</li>
```

And then add the label in the roomlistview.js as well:
```js
notificationCaptionText: t('spreed', 'Chat notifications'),
```

Comment (ma12-co) [reply js/views/templates/roomlistview.handlebars]: Right, I'm on it!

File: js/views/templates/roomlistview.handlebars [hunk 2]
```diff
@@ -43,7 +43,9 @@
 				<span>{{copyLinkText}}</span>
 			</button>
 		</li>
-		<li><div class="separator"></div></li>
+		<li><div class="separator"></div>
+		<li class="app-navigation-entry-menu-caption">{{notificationCaptionText}}</li>
+		</li>
```
Comment (nickvergessen) [inline js/views/templates/roomlistview.handlebars]: li inside li? that seems to be a mistake :thinking:

Comment (ma12-co) [reply js/views/templates/roomlistview.handlebars]: Yeah, It's the li from the separator, gonna fix it now!

**Comment by nickvergessen** [comment]
Tests seem unrelated

**Pull request merged**